src/app/InteractionModelEngine.cpp

```diff
@@ -1578,5 +1582,51 @@ void InteractionModelEngine::OnFabricRemoved(const FabricTable & fabricTable, Fa
     // the fabric removal, though, so they will fail when they try to actually send their command response
     // and will close at that point.
 }
+
+CHIP_ERROR InteractionModelEngine::ResumeSubscriptions()
+{
+#if CHIP_CONFIG_PERSIST_SUBSCRIPTIONS
+    SubscriptionResumptionStorage::SubscriptionIndex subscriberIndex;
+    CHIP_ERROR err = mpSubscriptionResumptionStorage->LoadIndex(subscriberIndex);
+    ChipLogProgress(InteractionModel, "%u subscriber nodes to resume.. (error %" CHIP_ERROR_FORMAT ")",
+                    static_cast<unsigned>(subscriberIndex.mSize), err.Format());
+    ReturnErrorOnFailure(err);
+
+    for (size_t i = 0; i < subscriberIndex.mSize; i++)
```

Comment:
Filed issue #24412 for this

Reply:
This has to be done with this PR since it involves the public API surface that will start being part of InteractionModelEngine. Making it at least indirected with a basic getter-with-index is trivial and can be done now. Iterator can be done later, but it will be less intrusive.

Reply:
This was done!